src/main/java/codeu/model/store/basic/ActivityStore.java

```diff
@@ -77,19 +77,14 @@ private ActivityStore(PersistentStorageAgent persistentStorageAgent) {
      * @param instant time that the Activity objects must be created on or after
      */
     public List<Activity> getActivitiesMadeOnOrAfter(Instant instant) {
-        int index = -1; // Index to take a sublist up to, exclusive
-        List<Activity> sortedActivities = getAllSortedActivities();
-        for (int i=0; i < sortedActivities.size(); i++) {
-            if (sortedActivities.get(i).getCreationTime().compareTo(instant) < 0) {
-                index = i;
-                break;
+        List<Activity> activitiesMadeOnOrAfter = new ArrayList<>();
+        for (int i=0; i < activities.size(); i++) {
```

Comment:
If you use getAllSortedActivities() here you can just break in the if-statement like before. Also using a enhanced for-loop here will allow you to stop using the #get method and the #size method.

Something like `for(Activity activity : getAllSortedActivies())`